nengo_ocl/raggedarray.py

```diff
@@ -33,8 +33,10 @@ class RaggedArray(object):
     in the same underlying buffer.
     """
 
-    def __init__(self, arrays, names=None, dtype=None, align=False):
-        arrays = [np.asarray(a) for a in arrays]
+    def __init__(self, arrays, names=None, dtype=None, align=False, order='C'):
+        assert order in 'CF'
```

Comment:
I think `assert order in ('C', 'F')` would be better, because 'CF' would also be accepted with what you have above, right?